drivers/gpu/drm/lima/lima_gem.c

```diff
@@ -34,7 +35,8 @@ struct lima_drm_priv *to_lima_drm_priv(struct drm_file *file)
 	return file->driver_priv;
 }
 
-static struct lima_bo *lima_gem_create_bo(struct drm_device *dev, u32 size, u32 flags)
+static struct lima_bo *lima_gem_create_bo(struct drm_device *dev, u32 size,
```

Comment:
Seems besides dma_addr, we have to handle a new resv obj arg. So I think we can move the amd_addr and resv init out of this function to the corresponding lima_gem_create_handle & lima_gem_prime_import_sg_table.